src/controller/AutoCommissioner.cpp

```diff
@@ -129,6 +129,26 @@ CHIP_ERROR AutoCommissioner::SetCommissioningParameters(const CommissioningParam
         mParams.ClearExternalBufferDependentValues();
     }
 
+    if (params.GetFailsafeTimerSeconds().HasValue())
+    {
+        mParams.SetFailsafeTimerSeconds(params.GetFailsafeTimerSeconds().Value());
```

Comment:
Why is this needed?  This has _already been done_ on line 125, no?

Reply:
You're right. I knew that copying was not possible in the case of Optional. I will delete this code.